src/song_utils.py

```diff
@@ -8,12 +8,45 @@
 from mutagen.id3 import ID3FileType, PictureType
 from mutagen.ogg import OggFileType
 from mutagen.wave import WAVE
-from nextcord import File
+from nextcord import Attachment, Embed, File, User
+from nextcord.utils import escape_markdown
 from PIL import Image, UnidentifiedImageError
 
 import config
 
 
+def embed_song(
+    message_content: str,
+    attachment_filepath: str,
+    attachment: Attachment,
+    user: User,
+    emoji: str,
+    jump_url: str,
+) -> Embed:
+
+    # Build and send "Now Playing" embed
```

Comment:
I think this comment might be outdated - `embed_song` doesn't send the embed, it only returns it.